content/en/user-guide/web-application/extensions-library/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Extensions Library"
+weight: 5
+description: Get started with Extensions Library to extend LocalStack by adding new services and features via Extension mechanism.
+---
+
+## Introduction
+
+LocalStack Extensions that allow developers to extend and customize LocalStack. A LocalStack Extension is a Python application that runs together with LocalStack within the LocalStack container.
```

Comment:
I see the reason behind it, but still would go with the regular word `extension` and thus not capitalize it.

Reply:
I'm ok with `Extensions Library` though. in the app we had similar thoughts when it comes to `auth tokens` or `Auth Tokens` for that matter but in the end settled with the former.